test-framework/src/main/java/com/datastax/oss/quarkus/test/SkipIfNotAmd64.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright DataStax, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.datastax.oss.quarkus.test;
+
+import java.lang.annotation.Retention;
+import java.lang.annotation.RetentionPolicy;
+import org.junit.jupiter.api.extension.ExtendWith;
+
+@Retention(RetentionPolicy.RUNTIME)
+@ExtendWith(CheckAmd64Condition.class)
```

Comment:
You could annotate this with `@java.lang.annotation.Inherited` and then use the annotation directly on `DseTestBase` instead of using it on each concrete subclass.

Reply:
Excellent point, I like that solution quite a bit better!  Changing now...

Reply:
Bah, my system clock was off (bad VM!) so the commit actually shows in history _before_ @adutra made his comment mentioning it.  [Relevant commit](https://github.com/datastax/cassandra-quarkus/pull/226/commits/e915f273bf96223fd1501919be240ee431300313) is above, tests for Java11 and Java17 running now.